ts/features/wallet/onboarding/privative/screens/search/LoadPrivativeSearch.tsx

```diff
@@ -36,9 +34,6 @@ const LoadPrivativeSearch = (props: Props): React.ReactElement | null => {
   const { privativeSelected } = props;
 
   if (privativeSelected === undefined) {
-    showToast(I18n.t("global.genericError"), "danger");
```

Comment:
Is it ok to give no UI feedback if `privativeSelected` is undefined?
(I could have a low context on this story)

Reply:
This screen is managed by `SearchPrivativeCardScreen`.

The case in which you arrive in this screen with `privativeSelected === undefined` is not expected, so this is an extra layer of safety.

I removed this functions in the component because they're already called by `SearchPrivativeCardScreen` with the same condition and the `SearchPrivativeCardScreen` is the only screen that use `LoadPrivativeSearch`.
https://github.com/pagopa/io-app/blob/326c075f1a8a3399dfa700f9aefb8b240e96fdbd/ts/features/wallet/onboarding/privative/screens/search/SearchPrivativeCardScreen.tsx#L74-L78